Assignments/Program.cs

```diff
@@ -1,3 +1,50 @@
-﻿using System;
+﻿// ---------------------------------------------------------------------------------------
+// Spark23 Assignments
+// Copyright (c) Metamation India.
+// ---------------------------------------------------------------------------------------
+// Program.cs
+// T16 - Longest Abcdedarian Word
+// ---------------------------------------------------------------------------------------
 
-Console.WriteLine("Hello, World!");
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Spark;
+public class T16 {
+   public static void Main () {
+      Console.WriteLine ("\x1B[4m" + "Longest Abcdedarian Finder:-" + "\x1B[0m");
+      List<string> words = new () { "apple", "first", "antwz", "efghijlk", "pqrst", "orange" };

```

Comment:
```suggestion
      List<string> words = new () { "apple", "firsty", "antwz", "efghijlk", "pqrst", "orange" };

```
I expect the output to be firsty. But not so.